<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphing Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="pushButton_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>581</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3066BE;
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: #A9A9A9; 
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusMinus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #090c9b;
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: #A9A9A9; 
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #090c9b;
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: #A9A9A9; 
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subtract">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #090c9b;
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: #A9A9A9; 
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>730</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #090c9b;
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: #A9A9A9; 
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>730</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3066BE;
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: #A9A9A9; 
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>730</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray; 
	background -color: #C0C0C0;
} 
QPushButton:pressed {
    background-color: #A9A9A9; 
 	border: 1px solid gray; 
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decimal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>730</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EqualSign">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>730</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>tan()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>sin()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>cos()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ArcTan">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Arctan()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ArcCos">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Arccos()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ArcSin">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Arcsin()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Shift">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3066BE;
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: #A9A9A9; 
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Shift</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cot">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>cot()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ArcCot">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Arccot()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_csc">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>csc()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ArcCsc">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Arccsc()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ArcSec">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Arcsec()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sec">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #3c3744;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>sec()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GreaterThan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pi">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>730</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>ฯ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exponent">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>a^(b)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LessThan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GreaterThanEqual">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>&gt;=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Y">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LessThanEqual">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>โ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Square">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>a^(2)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Euler">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NaturalLog">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>ln  log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LeftParenthesis">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>730</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RightParenthesis">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>730</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AbsoluteValue">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: #807f7f;
  border: 1px solid white;
}
QPushButton:pressed {
    background-color: #A9A9A9;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>|a|</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
